tome-app/.gitignore

```diff
@@ -4,21 +4,68 @@ logs
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
-pnpm-debug.log*
-lerna-debug.log*
-
-node_modules
-dist
```

Comment:
You want to ignore dist (or whatever your build folder is) as it would be build by running `npm run build`. Same with node modules